README.md

```diff
@@ -26,6 +26,34 @@ generate the `.yaml` entries here.
 
 ## Using this data
 
+### Marking specific attributes
+It can be helpful to know which specific code elements of a package are vulnerable and this is done by appending an attribute and list of module paths to that attribute to the OSV payload. Eg. 
+```
+{
+  attribute: "ImageFont",
+  modules: ["PIL"]
```

Comment:
Nitpick: JSON syntax since this'll be part of an OSV payload

```suggestion
  "attribute": "ImageFont",
  "modules": ["PIL"]
```

(and similar below)